packages/sirius-web/backend/sirius-web-application/src/main/resources/schema/siriusweb.graphqls

```diff
@@ -40,6 +41,24 @@ type ProjectTemplate {
   imageURL: String!
 }
 
+type ViewerLibrariesConnection {
+  edges: [ViewerLibrariesEdge!]!
+  pageInfo: PageInfo!
+}
+
+type ViewerLibrariesEdge {
+  node: Library!
+}
+
+type Library {
+  id: ID!
+  projectId: String!
```

Comment:
Don't just copy data from another type of the schema but use a reference instead. Something like `project: Project`. The goal of GraphQL is to build a graph after all.